.github/workflows/presubmit.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         if: ${{ github.event_name == 'workflow_dispatch' && inputs.debug_enabled }}
       - name: Integration test
         run: |
-          THAPI_BIN_DIR=./build/ici/bin bats integration_tests/
+          THAPI_INSTALL_DIR=./build/ici bats integration_tests/
```

Comment:
why? And if required can we do that in a separate PR?

Reply:
For the toggle integration tests, we need `THAPI_INC_DIR` and `THAPI_LIB_DIR` (to link against
`libThapiToogle.so` and include `thapi.h`). I could set them up separately but I think it is easier
to pass in the `THAPI_INSTALL_DIR` and then use that to find `THAPI_INC_DIR, THAPI_LIB_DIR`
and `THAPI_BIN_DIR`.